manifest.xml

```diff
@@ -9,5 +9,4 @@
   <logo>http://</logo>
   <depend package="base/cmake" />
   <depend package="base/types" />
-  <depend package="drivers/hidapi" />
```

Comment:
When you do e.g. an aup, you have a line looking like

operating system: ubuntu,debian - 16.04,16.04.1,lts,xenial,xerus

The first part (before the dash) are the OS names (comma separated, by order of priority)
The second part (after the dash) are the OS versions (comma separated, by order of priority)

You should be also able to see how autoproj resolves libhidapi for your OS by running `autoproj show libhhidapi`